react/features/app/components/App.native.js

```diff
@@ -149,6 +149,22 @@ export class App extends AbstractApp {
         }
     }
 
+    /**
+     * Creates an extra {@link ReactElement}s to be added (unconditionally)
+     * alongside the main element.
+     *
+     * @returns {ReactElement}
+     * @abstract
+     * @protected
+     */
+    _createExtraElement() {
```

Comment:
Let's just drop this and have web add the overlay container, there is no need for the extra hook, this abstraction has already been painful enough...